webcompat/views.py

```diff
@@ -143,6 +144,7 @@ def index():
 
 
 @app.route('/issues')
+@cache_policy(private=True, uri_max_age=86400)
```

Comment:
hmmm? :) I did change it already ;)
See https://github.com/webcompat/webcompat.com/pull/1335/files#diff-23e0a36bbee957147173901eb77913cbR463